sqlite-db/src/models.rs

```diff
@@ -338,3 +338,39 @@ impl From<Position> for model::Position {
         }
     }
 }
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct Identity(x25519_dalek::PublicKey);
+
+impl fmt::Display for Identity {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let hex = hex::encode(self.0.as_bytes());
```

Comment:
⚠️ Isn't that a breaking change because we would not write something hex encoded into the db that was previously not hex encoded?

Reply:
It shouldn't be a breaking change but let me double check this.

Reply:
It's not a breaking change. I just copied the existing display implementation from here: 
https://github.com/itchysats/itchysats/blob/8bb7d6dc524213f16a99617d16571f249ac4673d/model/src/lib.rs#L507-L513

Tested with mainnet DB locally as well and works.